crates/re_data_store/src/store_subscriber.rs

```diff
@@ -112,6 +112,20 @@ impl DataStore {
         })
     }
 
+    /// Passes a reference to the downcasted subscriber to the given `FnOnce` callback.
+    ///
+    /// Returns `None` if the subscriber doesn't exist or downcasting failed.
+    pub fn with_subscriber_once<V: StoreSubscriber, T, F: FnOnce(&V) -> T>(
```

Comment:
If `F` captures anything by value and then moves that value (like e.g. in `Caches::with_latest_at`) then it cannot be `FnMut` since it can only be called once.

I could make `with_subscriber` take a `FnOnce`, but then im preventing callers from actually working with `FnMut` closures.